From 264063de2bb541915aaaf81a66971c9eb7b34af8 Mon Sep 17 00:00:00 2001
From: Shreenidhi Shedi <shreenidhi.shedi@broadcom.com>
Date: Fri, 17 May 2024 22:59:48 +0530
Subject: [PATCH 28/39] ssh denial fix

This happens when pam-faillock is enabled.

type=AVC msg=audit(1716303120.620:282): avc:  denied  { read write } for  pid=1080 comm="sshd" name="root" dev="sda3" ino=655375 scontext=system_u:system_r:sshd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:var_log_t:s0 tclass=file permissive=1

Signed-off-by: Shreenidhi Shedi <shreenidhi.shedi@broadcom.com>
---
 policy/modules/services/ssh.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/services/ssh.te b/policy/modules/services/ssh.te
index 915118b..4e932e5 100644
--- a/policy/modules/services/ssh.te
+++ b/policy/modules/services/ssh.te
@@ -761,3 +761,5 @@ optional_policy(`
 	xserver_use_xdm_fds(ssh_agent_type)
 	xserver_rw_xdm_pipes(ssh_agent_type)
 ')
+
+allow sshd_t var_log_t:file { open read write };
-- 
2.45.2

